### ISSUE TITLE
1.0.9 breaks support for non-Windows OSs

### ISSUE THREAD

**Issue by gberardinelli** [chriskiehl/Gooey#773]
Hello,

I'm on the `1.0.9-release` branch looking into adding a feature, but this branch seems to have broken support for non-Windows operating systems in this commit:  https://github.com/chriskiehl/Gooey/commit/13a0c9e182b2787810073b10ed8f156b2ede59bf#diff-c9a4afaa360d6d71bcf32ad1253f3840ad139f43e88733ba3a11b3927294d9fdL87-L92

This is because `subprocess.CREATE_NEW_PROCESS_GROUP` is a Windows only constant, as shown in the [docs](https://docs.python.org/3/library/subprocess.html#windows-constants) and confirmed in the [cpython source](https://github.com/python/cpython/blob/f1a58441eea6b7788c64d03a80ea35996301e550/Lib/subprocess.py#L78-L88).

The GUI comes up initially, but hitting "start" results in an exception:

```
Traceback (most recent call last):
  File "/home/gberardinelli/Scripts/Gooey/gooey/gui/processor.py", line 75, in run
    creationflags=subprocess.CREATE_NEW_PROCESS_GROUP)
AttributeError: module 'subprocess' has no attribute 'CREATE_NEW_PROCESS_GROUP'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/gberardinelli/venv/gooey/lib/python3.6/site-packages/wx/core.py", line 3407, in <lambda>
    lambda event: event.callable(*event.args, **event.kw) )
  File "/home/gberardinelli/Scripts/Gooey/gooey/gui/containers/application.py", line 93, in onStart
    self.clientRunner.run(self.buildCliString())
  File "/home/gberardinelli/Scripts/Gooey/gooey/gui/processor.py", line 81, in run
    creationflags=subprocess.CREATE_NEW_PROCESS_GROUP
AttributeError: module 'subprocess' has no attribute 'CREATE_NEW_PROCESS_GROUP'
```

I can work on a PR but it may be some time before I can test on Windows, so I figured I'd raise it here first.

For completeness, I'm on Linux Mint 19.3 / Python 3.6 but of course this should apply to anything that's not Windows.

Cheers

**Comment by chriskiehl** [comment]
Ah, thanks for the report. I'll have to revisit. ^_^ 

FYI, if you're planning on adding something to 1.0.9, I'd suggest holding off a bit. [1.0.9 changed directions](https://github.com/chriskiehl/Gooey/issues/755) and a lot of Gooey's guts have been entirely removed and replaced in support of the new functionality -- its basically turned into a _light_ rewrite of most of the UI layers. So, 1.0.9 as it exists is no longer a good view of the project.  

I'd vote for opening an issue with the feature you want to add so we can make sure it fits prior to dumping time into the work.

**Comment by chriskiehl** [comment]
This was corrected in 1.2.0.
